tests/test_integration.py

```diff
@@ -33,6 +35,12 @@
     webdriver = None
 
 
+try:
+    from django.test import AsyncRequestFactory
+except ImportError:
+    AsyncRequestFactory = None
```

Comment:
`AsyncRequestFactory` is defined in Django >= v3.1 so the `ImportError` should not happen on all supported versions of Django.

Just modify line 16 to read `from django.test import AsyncRequestFactory, RequestFactory`